scalyr_agent/util.py

```diff
@@ -156,6 +156,63 @@ def json_decode(text):
     return _json_decode(text)
 
 
+def json_scalyr_request_encode(value, output=None, use_length_prefix_string=False):
```

Comment:
Do you have my latest changes?  I believe this function should be deleted.. at the least, let's put in a TODO to remove it.

Reply:
Yes. Changes have been merged now.